pyrolite/plot/spider.py

```diff
@@ -209,7 +209,7 @@ def spider(
             )
             # do these need to be ravelled?
             ax.scatter(
-                indexes.ravel(), arr.ravel(), c=scattercolor, **{"zorder": 2, **s_kw}
+                indexes.ravel(), arr.ravel(), color=scattercolor, **{"zorder": 2, **s_kw}
```

Comment:
I'm not sure this one needs to change, but will verify that this doesn't cause any unintended consequences after merge.

Reply:
I think this one was to remove a deprecation warning.